lib/src/base-styles/_nx-colors.scss

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+:root {
+  --nx-color-border-default: #{$nx-indigo-100};
+  --nx-color-border-subsection: #{$nx-grey-200};
+  --nx-color-border-form-element: #{$nx-grey-1100};
+  --nx-color-border-form-element-pristine: #{$nx-indigo-100};
+  --nx-color-border-disabled: #{$nx-grey-500};
+  --nx-color-border-focus: #{$nx-teal-300};
+
+  --nx-color-box-shadow-focus: #{transparentize($nx-teal-300, 0.7)};
```

Comment:
I'd group this with the other interactive states below and name it similarly.

Reply:
> I'd group this with the other interactive states below and name it similarly.
>
> --nx-color-element-focus-glow?

Why? That's less consistent than what I have now.  Especially renaming it to "glow" rather than an actual CSS property name

Reply:
> > I'd group this with the other interactive states below and name it similarly.
> > --nx-color-element-focus-glow?
> 
> Why? That's less consistent than what I have now. Especially renaming it to "glow" rather than an actual CSS property name

I'm not wedded to "glow" but the idea is to name it according to the effect achieved, not how the effect was achieved.

Reply:
> I'm not wedded to "glow" but the idea is to name it according to the effect achieved, not how the effect was achieved.

How about just "shadow" instead of "box-shadow"? Since it can also be used as a drop-shadow

Reply:
> > I'm not wedded to "glow" but the idea is to name it according to the effect achieved, not how the effect was achieved.
> 
> How about just "shadow" instead of "box-shadow"? Since it can also be used as a drop-shadow

Sure, that makes sense.

Reply:
> Sure, that makes sense.

2cddcd6d